source/gameengine/Physics/Bullet/CcdPhysicsController.cpp

```diff
@@ -1600,6 +1600,9 @@ PHY_IPhysicsController *CcdPhysicsController::GetReplicaForSensors()
 
 			default:
```

Comment:
@youle31 : Could you check that `cinfo.m_collisionShape = m_shapeInfo->CreateBulletShape(m_cci.m_margin, m_cci.m_bGimpact, !m_cci.m_bSoft);` can do the full work ?